apps/pie-docs/src/components/checkbox/overview/overview.md

```diff
@@ -0,0 +1,299 @@
+      width: "94px",
+      alt: "Checkbox component in an unselected state."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-partially-selected.svg",
+      width: "154px",
+      alt: "Checkbox component in a partially selected state."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Interactions
+
+Users can trigger the checkbox directly or by clicking the checkbox label. Having all regions interactive creates a more accessible click target.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-interaction.svg",
+      width: "70px",
+      alt: "An illustration represents that users can trigger the checkbox directly."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-label-interaction.svg",
+      width: "70px",
+      alt: "An illustration represents that users can trigger the checkbox by clicking the checkbox label. "
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Interactive states
+
+Outlines the atomic level interactive elements for the component.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h4>Default</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-default.svg",
+      width: "70px",
+      alt: "Checkbox component in a default state."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Hover</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-hovered.svg",
+      width: "70px",
+      alt: "Checkbox component when hovered over."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Active</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-active.svg",
+      width: "70px",
+      alt: "Checkbox component in an active state."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Focus</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-focused.svg",
+      width: "70px",
+      alt: "Checkbox component when it has focus."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Disabled</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-disabled.svg",
+      width: "70px",
+      alt: "Checkbox component in a disabled state."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Error</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox/checkbox-error.svg",
+      width: "112px",
+      alt: "Checkbox state example showing an error."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Examples
+
+Outlines the atomic level interactive elements for the component.
+
```

Comment:
I think I remember there was a discussion saying that this piece of text only belongs in the "Interactive states" section above. @tingting-lau?

```suggestion
```

Reply:
That is correct :) Thanks Xander.